### PR TITLE
Google maps listing search backup

### PR DESCRIPTION
**Pull request by Pochetes**
## Address Inputs
- user is now able to add their own address powered by Google's Autocomplete widget can search for any address.
- when creating pool listing, user can list their exact address of the pool
- exact address is hidden from users, only displays city and state in the listing menu and details

user can search for their own address when signing up
register

they can also search for their exact pool location when they create a new listing
createpool

## Pool Listing Search
- user can filter pool locations (currently by city) and will conditionally display the searched listings

search

### CODE REVIEW

**Comment by Pochetes** [comment]
>Good thinking on hiding the address to users. 
>
>I think your previous diff is in this PR. make sure to not include it!
>
>_Originally posted by @roeshanigos in https://github.com/Facebook-University-Projects/Swimster/pull/15#pullrequestreview-1063898411_

Per this [thread](https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch#:~:text=This%20happens%20with%20GitHub%20when,merges%20from%20the%20target%20branch.), GitHub does not rebase the commit history automatically in pull requests. Therefore, creating a new diff worked.

File: swimster-ui/src/components/PoolMap/PoolMap.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { GoogleMap, LoadScript } from '@react-google-maps/api'
+
+const mapContainerStyle = {
+    width: '100%',
+    height: '100%',
+}
+
+const center = {
```
Comment (roeshanigos) [inline swimster-ui/src/components/PoolMap/PoolMap.jsx:9]: where is this? good to comment that in

**Review by roeshanigos — APPROVED** [review]
nice! another design enhancement - make the search also have a rendered suggestion list!

File: swimster-ui/src/components/PoolMap/PoolMap.jsx (hunk 1 above)
Comment (Pochetes) [reply swimster-ui/src/components/PoolMap/PoolMap.jsx:9]: oh, these are the attributes of the map. will comment that

**Pull request merged**